### ISSUE TITLE
No jei with JEI?

### ISSUE THREAD

**Issue by bookerthegeek** [The-Acronym-Coders/B.A.S.E#45]
Interesting, was reading my server log and found this during startup...
```
[Client thread/FATAL] [base]: Module JEI did not load due to issue: Mod with modid: jei was not found
```
Interestingly enough, JEI is installed.
Also Interesting to note that the Mod ID for JEI is `JEI` and not `jei`, No idea if that matters or not.

JEI: 1.10.2-3.14.5.406
Base: 1.10.2-1.2.3
Forge: 12.18.3.2234
MC: 10.2

**Comment by SkySom** [comment]
Oops that one is fixed in dev, didn't realize it exists in prod

**Comment by Lanse505** [comment]
@SkySom have we pushed a new build to fix this?

**Comment by SkySom** [comment]
Nope, I've been trying to wait to release with Content Tweaker, gonna push out a hotfix instead

**Comment by SkySom** [comment]
Fixed in 1.3 (On curse already)
